compiler/rustc_driver_impl/src/signal_handler.rs

```diff
@@ -6,6 +6,18 @@ use std::{fmt, mem, ptr, slice};
 
 use rustc_interface::util::{DEFAULT_STACK_SIZE, STACK_SIZE};
 
+/// Signals that represent that we have a bug, and our prompt termination has
+/// been ordered.
+const KILL_SIGNALS: [(libc::c_int, &str); 7] = [
+    (libc::SIGILL, "SIGILL"),
+    (libc::SIGTRAP, "SIGTRAP"),
```

Comment:
this signal is for the usage of a debugger and we should not pollute their execution with our handler.